### PR TITLE
JAMES-3034 Disable test case has been passing because one of SMTPMessageSender fault

### PR DESCRIPTION
**Pull request by trantienduchn**
Before that, the message is altered by SMTPMessageSender, so Unicode
characters are altered in '?', it leads to 7bit transfer-encoding.
So the RemoteDelivery doesn't break the DKIM signature

### CODE REVIEW

File: server/mailet/integration-testing/src/test/java/org/apache/james/mailets/RemoteDeliveryDKIMIntegrationTest.java [hunk 1]
```diff
@@ -130,9 +129,13 @@ void tearDown() {
     class WhenEnable8BitMime {
         @Disabled("JAMES-3016 assertion failed:" +
             "org.apache.james.jdkim.exceptions.PermFailException: Computed bodyhash is different from the expected one")
```
Comment (rouazana) [inline server/mailet/integration-testing/src/test/java/org/apache/james/mailets/RemoteDeliveryDKIMIntegrationTest.java]: why JAMES-3016 is closed if it is not fixed?

Comment (Arsnael) [reply server/mailet/integration-testing/src/test/java/org/apache/james/mailets/RemoteDeliveryDKIMIntegrationTest.java]: My bad... Reopening it then

Comment (trantienduchn) [reply server/mailet/integration-testing/src/test/java/org/apache/james/mailets/RemoteDeliveryDKIMIntegrationTest.java]: It's fixed, by removing `allow8bitmime` in the code base. In this test, I want to show people the issue can appear when you intentionally enable `allow8bitmime` in your James server

Comment (rouazana) [reply server/mailet/integration-testing/src/test/java/org/apache/james/mailets/RemoteDeliveryDKIMIntegrationTest.java]: oh right sorry I read to fast.
ok to merge this as is.
but for me as it is an expected behavior the test should not be disabled, it just should be negated.

Comment (trantienduchn) [reply server/mailet/integration-testing/src/test/java/org/apache/james/mailets/RemoteDeliveryDKIMIntegrationTest.java]: Damn I don't have the right, please help me close it

**Comment by nvduc91** [comment]
Read it.

**Comment by trantienduchn** [comment]
```
[d2611a4bc9a92dbdd9c3c4db5eb3ff7d31b84e8f] [ERROR] Errors: 
[d2611a4bc9a92dbdd9c3c4db5eb3ff7d31b84e8f] [ERROR]   MemoryDumbBlobStoreTest.mixingSaveReadAndDeleteShouldReturnConsistentState » Execution
```

**Comment by chibenwa** [comment]
Please test the two build fix PR together

**Comment by trantienduchn** [comment]
> Please test the two build fix PR together

It's #3098

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**